README.md

```diff
@@ -568,59 +616,58 @@ document.addEventListener("pjax:success", whenDOMReady)
 
 _Note: Don't create the Pjax instance in the `whenDOMReady` function._
 
-For my concern and usage, I `js-Pjax`-ify all body children, including stuff like navigation and footer (to get navigation state easily updated).
+For my concern and usage, I `js-Pjax`-ify all body children, including stuff like navigation and footer (to update navigation state).
 
 The attached behavior is re-executed each time a page is loaded, like in the snippet above.
 
-If you want to just update a specific part (it's totally a good idea), you can just
-add the DOM-related code in a function and re-execute this function when "pjax:success" event is fired.
+If you want to just update a specific part (it's totally a good idea), you can add the DOM-related code in a function and re-execute this function when the `pjax:success` event is fired.
```

Comment:
_'If you want to just update a specific part of the page (which is a good idea)...'_